graph/xml_parser.py

```diff
@@ -6,6 +6,14 @@
 import uuid
 import graph.oval_graph
 
+global ns
```

Comment:
Global here makes no difference. You use `global` e.g. inside functions when you want to modify the global object. Read-only access works without `global`.